src/shared/components/tabbar/tabbar.css.ts

```diff
@@ -0,0 +1,50 @@
+import { style } from "@vanilla-extract/css";
+
+import { themeVars } from "@shared/styles/theme.css";
+
+export const wrapper = style({
+  position: "fixed",
+  bottom: 0,
+  left: "50%",
+  transform: "translateX(-50%)",
+  width: "100%",
+  minWidth: "var(--min-width)",
+  maxWidth: "var(--max-width)",
+  zIndex: 99,
+});
+
+// TabBar 레이아웃
+export const container = style({
+  width: "100%",
+  boxSizing: "border-box",
+  margin: 0,
+  padding: "0.8rem 1.6rem 1.2rem",
+  listStyle: "none",
+
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "space-between",
```

Comment:
디자인 대로라면
```suggestion
  gap: '2.3rem'
```
이게 맞는것 같네용용